Update annotations.xml

pnzlqa111110 opened the pull request:
adding a few more

pnzlqa111110 committed:
Update annotations.xml

adding more options

nicolas-raoul commented on annotations.xml:
```diff
@@ -723,21 +724,27 @@
 <Annotation about="*.wfae.org/*"><Label name="_include_"></Label></Annotation>
 <Annotation about="*.wfae.org/opinion/*"><Label name="_exclude_"></Label></Annotation>
 <Annotation about="*.whyy.org/*"><Label name="_include_"></Label></Annotation>
+    <!-- Members of the Global Investigative Journalism Network (all 3 rated members deemed generally reliable) - 200+ members can be found here: https://gijn.org/membership/gijn-global-network/ -->
```
Did you mean to add 3 after this line?

pnzlqa111110 replied:
ah no as those 3 members are included above in the Perennial Sources section - should’ve clarified that it is just a placeholder for more potential outlets to add

pnzlqa111110 committed:
Update annotations.xml

clarifying where the perennial sources are and adding five for clarity so as not to conflict with changes proposed more recently

pnzlqa111110 reviewed (commented):
Hopefully this addresses the confusion and clears the way for future commits without conflicts

nicolas-raoul commented:
What is the status for this? :-)

pnzlqa111110 commented:
> What is the status for this? :-)

This one is good-to-go

pnzlqa111110 committed:
Update annotation comment for Global Investigative Journalism Network
